### PR TITLE
created inferface for Randao.sol. I followed the same path for import…

### PR DESCRIPTION
**Pull request by BarneyChambers**
created inferface for Randao.sol. I followed the same path for importing IAxiom.sol as in the Randao.sol contract, but I believe it might be incorrect, since is seems like the appropriate files are currently here: https://github.com/axiom-crypto/axiom-v1-contracts/contracts/interfaces/IAxiomV1.sol

### CODE REVIEW

**Comment by jonathanpwang** [comment]
in `remappings.txt` we have a remapping
```
axiom-contracts/=../lib/axiom-v1-contracts/
```
so your import looks OK.

File: randao/src/interfaces/IRandao.sol [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
```
Comment (jonathanpwang) [inline randao/src/interfaces/IRandao.sol:2]: change to 0.8.19, we aren't sure higher versions will work

File: randao/src/interfaces/IRandao.sol [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import {IAxiomV1} from "axiom-contracts/contracts/interfaces/IAxiomV1.sol";
+
+interface IRandao {
+    function prevRandaos(uint32 blockNumber) external view returns (uint256);
+    function axiomAddress() external view returns (address);
+    function mergeBlock() external view returns (uint32);
+
+    function updateAxiomAddress(address _axiomAddress) external;
```
Comment (jonathanpwang) [inline randao/src/interfaces/IRandao.sol]: not sure this function will be useful in the interface, since it's for the `onlyOwner` role

**Review by jonathanpwang — COMMENTED** [review]
Some minor nits, otherwise looks good. Thanks!

File: randao/src/interfaces/IRandao.sol (hunk 1 above)
Comment (BarneyChambers) [reply randao/src/interfaces/IRandao.sol:2]: lgtm :)

File: randao/src/interfaces/IRandao.sol (hunk 2 above)
Comment (BarneyChambers) [reply randao/src/interfaces/IRandao.sol]: That's a good point, I've moved it in my latest commit

**Comment by BarneyChambers** [comment]
Please let me know if you want me to make any more amendments :)

**Review by jonathanpwang — APPROVED** [review]
LGTM

**Pull request merged**